src/main/java/com/github/maleksandrowicz93/cqrsdemo/student/AddStudentCommandHandler.java

```diff
@@ -0,0 +1,40 @@
+package com.github.maleksandrowicz93.cqrsdemo.student;
+
+import com.github.maleksandrowicz93.cqrsdemo.student.dto.SaveStudentRequest;
+import com.github.maleksandrowicz93.cqrsdemo.student.dto.StudentDto;
+import com.github.maleksandrowicz93.cqrsdemo.student.exception.InvalidCredentialsException;
+import com.github.maleksandrowicz93.cqrsdemo.student.exception.StudentAlreadyExistsException;
+import lombok.RequiredArgsConstructor;
+import lombok.extern.log4j.Log4j2;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.security.crypto.password.PasswordEncoder;
+import org.springframework.stereotype.Component;
+
+@Log4j2
+@Component
+@RequiredArgsConstructor
+class AddStudentCommandHandler {
+
+    final StudentRepository studentRepository;
```

Comment:
Zamiast finali we wszystkich polach możesz dać `@FieldDetauls(makeFinal = true)` nad klasą